### DISCUSSION TITLE
Unable to download Keycloak 15.0.8

### DISCUSSION THREAD

**Discussion by somesh2012** [keycloak/keycloak#22056] [Feedback]
Hello everyone,

Greetings !!!
I tried to search a lot but was unable to find the download link for Keycloak version 15.0.8.
Version 15.0.2 and other versions are available but specifically, v15.0.8 is not available to download.
I can see Red Hat SSO is based on Keycloak version 15.0.8 but can't find Keycloack with version 15.0.8 anywhere.
Can anyone please help me to find the download link?

Thanks,
Somesh

**Comment by darius-m** [comment]
I don't think there was ever a version 15.0.8 of Keycloak. If I remember correctly, it went from 15.0.2 to 15.1 and then 15.1.1, followed by 16. RHSSO is an enterprise solution that is based on Keycloak. I assume it got to version 15.0.8 because of some backported patches by RedHat, but Keycloak itself does not have support for older versions.

**Reply by vmuzikar** [reply]
Yes, that's correct.

**Reply by somesh2012** [reply]
I have seen this version number in RH SSO v7.5 which is based on Keycloack 15.0.8, that's why I'm looking for this version.
By the way, thanks for the reply,
